### PR TITLE
ISPN-11959 Add custom NonBlockingStore server test

### PR DESCRIPTION
**Pull request by wburns**
https://issues.redhat.com/browse/ISPN-11959

### CODE REVIEW

File: server/tests/src/test/java/org/infinispan/server/persistence/CustomStoreIT.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+/**
+ * @author William Burns &lt;wburns@redhat.com&gt;
+ * @since 11.0
+ **/
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+      CustomStoreOperations.class,
+})
+public class CustomStoreIT {
```
Comment (ryanemerson) [inline server/tests/src/test/java/org/infinispan/server/persistence/CustomStoreIT.java:21]: Is there a need to create a Suite here? It seems overkill IMO.

Comment (wburns) [reply server/tests/src/test/java/org/infinispan/server/persistence/CustomStoreIT.java:21]: Fine by me, is there an example of one where it isn't done by a test Suite?

Comment (wburns) [reply server/tests/src/test/java/org/infinispan/server/persistence/CustomStoreIT.java]: Done

**Comment by ryanemerson** [comment]
Thanks @wburns

**Pull request merged**